graylog-plugin-parent/pom.xml

```diff
@@ -42,6 +42,7 @@
 
     <properties>
         <graylog.plugin-dir>/usr/share/graylog-server/plugin</graylog.plugin-dir>
+        <graylog.version>${project.parent.version}</graylog.version>
```

Comment:
This is a false friend as `${project.parent.version}` might not be the `graylog-plugin-parent` POM (e. g. in case a plugin itself is using submodules). It's resolved later in the Maven build process. 😞